### PR TITLE
add Missile Launcher and Super Missile Expansion offworld model for dread

### PR DESCRIPTION
**Pull request by steven11sjf**
***waiting on ODR release***

- adds the Missile Launcher model to Metroid Dread's pickup database
- adds dread's Missile Launcher model to am2r, prime, echoes, and super metroid's pickup databases

### CODE REVIEW

**Comment by Miepee** [comment]
tests fail

File: randovania/games/am2r/pickup_database/pickup-database.json [hunk 1]
```diff
@@ -401,7 +401,7 @@
             "offworld_models": {
                 "prime1": "Missile",
                 "prime2": "MissileLauncher",
-                "dread": "powerup_icemissile"
+                "dread": "powerup_missile"
```
Comment (Miepee) [inline randovania/games/am2r/pickup_database/pickup-database.json:404]: didnt you also already have a super launcher? or do you wanna put that off to another PR?

Comment (duncathan) [reply randovania/games/am2r/pickup_database/pickup-database.json:404]: does it not already use the vanilla super model?

Comment (Miepee) [reply randovania/games/am2r/pickup_database/pickup-database.json:404]: im dumb, meant super expansion

Comment (steven11sjf) [reply randovania/games/am2r/pickup_database/pickup-database.json:404]: nope. I still need to get the green M10 tank created. and merged into odr. probably gonna be another  PR unless dunc wants me to bundle everything into one.

**Comment by steven11sjf** [comment]
added the super missile expansion to this one. should be good once CI finishes.

File: CHANGELOG.md [hunk 2]
```diff
@@ -33,6 +33,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Dread
 
+- Added: New Missile Launcher model for Prime, Echoes, and AM2R multiworld pickups
+- Added: New Super Missile Expansion model for AM2R multiworld pickups
+- Fixed: Text is patched in all languages, not just English.
```
Comment (Miepee) [inline CHANGELOG.md:38]: > Use hash + layout UUID check

From the ODR changelog should also be mentioned. The human readable tldr, is that if you try to play different worlds from the same session (i.e if you have dread-starter and dread-expert), that the game now will also tell you that the saves are incompatible with each other


> Fix wide shields breaking from wave 

This should also be mentioned.

File: CHANGELOG.md [hunk 3]
```diff
@@ -33,6 +33,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Dread
 
+- Added: New Missile Launcher model for Prime, Echoes, and AM2R multiworld pickups
```
Comment (Miepee) [inline CHANGELOG.md:36]: dreads local missile launcher was also changed, no?

File: CHANGELOG.md [hunk 4]
```diff
@@ -33,6 +33,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Added: New Super Missile Expansion model for AM2R multiworld pickups
```
Comment (Miepee) [inline CHANGELOG.md:37]: You don't need to mention them in the changelog, but can you change the models for MSR too?

**Review by Miepee — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 3 above)
Comment (Miepee) [reply CHANGELOG.md:36]: Ah, right. Forgot about that.

**Review by Miepee — APPROVED** [review]
